### DISCUSSION TITLE
DataTable Filter "menu" exception:

### DISCUSSION THREAD

**Discussion by bhaidar** [primefaces/community#310] [PrimeVue]
Hi,
When using the filter display as a menu, I get exceptions on the Name column when I choose anything else but `contains`.

Is there a particular setup I should do? I disabled multiple filters on the Name field, I just want one filter but use any operator.

Here's a workable example:
https://codesandbox.io/p/sandbox/primevue-demo-forked-hl68j9?file=%2Fsrc%2FApp.vue%3A1%2C1-2%2C1

Here's the exception:
```
Uncaught TypeError: can't access property 0, _filters[this.field].constraints is undefined
    onMenuMatchModeChange datatable.esm.js:2432
    onUpdateModelValue datatable.esm.js:2828
    callWithErrorHandling runtime-core.esm-bundler.js:158
    callWithAsyncErrorHandling runtime-core.esm-bundler.js:166
    emit runtime-core.esm-bundler.js:669
    updateModel dropdown.esm.js:787
    onOptionSelect dropdown.esm.js:410
    onClick dropdown.esm.js:1169
    callWithErrorHandling runtime-core.esm-bundler.js:158
[datatable.esm.js:2432:6](https://hl68j9-5173.csb.app/node_modules/primevue/datatable/datatable.esm.js)
```

**Comment by tugcekucukoglu** [comment]
Hi,

Could you try to add `constraint` option? For example 

`name: { operator: FilterOperator.AND, constraints: [{ value: null, matchMode: FilterMatchMode.STARTS_WITH }] },`

**Reply by bhaidar** [reply]
Hi @tugcekucukoglu, thanks for your response.

I want to use a single filter, not more. So if I add the FilterOperator.ADD the user can add multiple filters.

Thanks
